app/assets/stylesheets/bento.scss

```diff
@@ -376,3 +378,28 @@ $z-depth-way-front:     1000;
 .about-bento {
   margin: 4rem 0;
 }
+
+.title-area {
+  margin-top: 1.5rem;
+  margin-bottom: 1rem;
+}
+
+.title.has-subline {
+  margin-bottom: .1em;
+  padding-bottom: .1em;
+}
+
+.subline {
+  font-size: $fs-xsmall;
+}
+
+.nav .current {
+  font-weight: $fw-bold;
+}
+
+// changes for testing - TODO: remove
```

Comment:
i need that to hide the source when testing - i think it's helpful for anyone developing to have that handy so I put a TODO on it and am adding it to the repo. I'll remove it when we don't need source any more for development. Cool?